shared/src/persistence/dynamo/cases/updateCase.test.js

```diff
@@ -460,6 +502,208 @@ describe('updateCase', () => {
     });
   });
 
+  describe('archivedCorrespondences', () => {
+    it('adds an archived correspondence to a case when archivedCorrespondences is an empty list', async () => {
+      await updateCase({
+        applicationContext,
+        caseToUpdate: {
+          archivedCorrespondences: [
+            {
+              archived: true,
+              documentId: mockCorrespondenceId,
+              documentTitle: 'My Correspondence',
+              filedBy: 'Docket clerk',
+              userId: 'user-id-existing-234',
+            },
+          ],
+          docketNumber: '101-18',
+          docketNumberSuffix: null,
+          status: CASE_STATUS_TYPES.generalDocket,
+        },
+      });
+
+      expect(
+        applicationContext.getDocumentClient().delete,
+      ).not.toHaveBeenCalled();
+      expect(applicationContext.getDocumentClient().put).toHaveBeenCalled();
+      expect(
+        applicationContext.getDocumentClient().put.mock.calls[0][0].Item,
+      ).toMatchObject({
+        pk: 'case|101-18',
+        sk: `correspondence|${mockCorrespondenceId}`,
+        userId: 'user-id-existing-234',
+      });
+    });
+
+    it('adds an archived correspondence to a case when archivedCorrespondences has entries', async () => {
+      addArchivedCorrespondences();
+
+      await updateCase({
+        applicationContext,
+        caseToUpdate: {
+          archivedCorrespondences: [
+            {
+              archived: true,
+              documentId: mockCorrespondenceId,
+              documentTitle: 'My Correspondence',
+              filedBy: 'Docket clerk',
+              userId: 'user-id-existing-234',
+            },
+            {
+              archived: true,
+              documentId: 'archived-correspondence-id-existing-123',
+              documentTitle: 'My Correspondence',
+              filedBy: 'Docket clerk',
+              userId: 'user-id-existing-234',
+            },
+          ],
+          docketNumber: '101-18',
+          docketNumberSuffix: null,
+          status: CASE_STATUS_TYPES.generalDocket,
+        },
+      });
+
+      expect(
+        applicationContext.getDocumentClient().delete,
+      ).not.toHaveBeenCalled();
+      expect(applicationContext.getDocumentClient().put).toHaveBeenCalled();
+      expect(
+        applicationContext.getDocumentClient().put.mock.calls[0][0].Item,
+      ).toMatchObject({
+        pk: 'case|101-18',
+        sk: `correspondence|${mockCorrespondenceId}`,
+        userId: 'user-id-existing-234',
+      });
+    });
+  });
+
+  describe('correspondence', () => {
+    it('adds a correspondence to a case when correspondence is an empty list', async () => {
```

Comment:
Similar to https://github.com/ustaxcourt/ef-cms/pull/364/files#r475987576 above - is `correspondence` supposed to be an empty list in this test? It doesn't appear to be in the `updateCase` call below.